elk/extraction/extraction_main.py

```diff
@@ -95,10 +102,14 @@ def extract(args, split: str):
     print("Loading datasets")
     silence_datasets_messages()
 
+    maybe_barrier()  # Not strictly necessary but makes the output cleaner
     extract(args, "train")
+    maybe_barrier()
+    print("wow")
```

Comment:
I guess the wow thing shoudld be removed (?)

Reply:
oh sorry that was a stupid debugging thing LOL